letsdrill.rb

```diff
@@ -1,24 +1,32 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+if integer >= 90
+  return "A"
+elsif integer >= 80
+  return "B"
+elsif integer >= 70
+  return "C"
+elsif integer >=60
+  return "D"
+else
+  return "F"
+end
```

Comment:
`return` is optional in ruby, and the standard practice is to omit it, except for 'early' returns.  Also, looks like the indentation here and and in the next method is a bit off - code inside a method should be indented two spaces.